frontend/src/components/UserUpdate.vue

```diff
@@ -24,6 +35,12 @@ export default {
     UserDetails,
     UserUpdateRoles
   },
+  data() {
+    return {
+      loading: true,
+      boilerplate: false,
```

Comment:
I tried to use these values directly in the attribute instead of binding them, but it broke the animation. I don't know why.